src/components/30-organisms/modal/index.scss

```diff
@@ -21,7 +21,7 @@
     flex-flow: column;
   }
 
-  &__upper-close-container {
+  &__upper-close-container {                                                                                              
```

Comment:
What happened here?

Reply:
It seems that I made a mistake copying code from it... Is there a way to undo this commit?